concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -39,6 +40,12 @@ prepare_environment() {
   cf_graphite_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.graphite.version "${cf_manifest_dir}/055-graphite.yml")
   cf_grafana_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.grafana.version "${cf_manifest_dir}/055-graphite.yml")
 
+  if ${COMMIT_VERIFICATION} ; then
```

Comment:
I don't like testing for empty strings, as you could then set SKIP_COMMIT_VERIFICATION=false or SKIP_COMMIT_VERIFICATION=no - and it would actually mean true/yes.